### DISCUSSION TITLE
New cosmetic problems

### DISCUSSION THREAD

**Discussion by Svardsten53** [mcguirepr89/BirdNET-Pi#336] [Q&A] [answered]
I have got a couple of cosmetic problems on one of my two installations. Both are updated just minutes ago, but shows different content on a couple of pages.

On "Today's Detections" I'm getting very big pictures of the birds.

![Skärmbild 2022-05-29 215249](https://user-images.githubusercontent.com/62435779/170889723-14c4a31e-163d-4f11-acdd-923c98b3a64f.png)

On "Best Recordings" I'm getting the text "Best Recordings for Other Species" out to the left instead of centered in a ribbon with lighter green color.

![Skärmbild 2022-05-29 215437](https://user-images.githubusercontent.com/62435779/170889800-f226d35d-f4e6-4aaf-9e26-6a1f65ddb3f4.png)

On my other installtion everything looks perfect. Please note that this was also there before translating the pages. I have tried copying the right page and style.css from the other installation without success.

**Comment by ehpersonal38** [comment, marked as answer]
I'd recommend to clear your browser cache and the issue should be resolved.

**Reply by Svardsten53** [reply]
Thanks! Ctrl-F5 solved it easily. Shame on me.  :angry:

**Reply by ehpersonal38** [reply]
No worries! I think because you have two installations your browser got confused... or something. Hope you enjoy the new updates! 🐦

**Reply by Svardsten53** [reply]
I really do! Thanks to everyone involved.
